src/txdb.cpp

```
@@ -313,6 +313,35 @@ bool CBlockTreeDB::ReadAddressIndex(uint160 addressHash, AddressType type,
+        pcursor->Next();
+    }
+
+    size_t counter = 0;
+    for (auto& itr : addrMap) {
+        if (itr.second > 0) {
+            ++counter;
+        }
+    }
+
+    return counter;
+}
```

cppcheck (2.10-2)

[performance] Variable 'type' is assigned in constructor body. Consider performing initialization in initialization list.

(useInitializationList)